components/create_team/index.ts

```diff
@@ -41,4 +46,4 @@ function mapStateToProps(state) {
 export default compose(
     connect(mapStateToProps),
     withUseGetUsageDelta,
-)(CreateTeam);
+)(CreateTeam) as React.FunctionComponent<Props & RouteComponentProps & {usageDeltas: CloudUsage}>;
```

Comment:
The props contains all the above types already
```suggestion
)(CreateTeam) as React.ComponentType<Props>;
```

Reply:
Thanks for the suggestion. I have made the change